definitions/procedures/repositories/index_katello_repositories_container_metadata.rb

```diff
@@ -0,0 +1,17 @@
+module Procedures::Repositories
+  class IndexKatelloRepositoriesContainerMetatdata < ForemanMaintain::Procedure
+    metadata do
+      description 'Import container manifest metadata into Katello'
+      confine do
+        feature(:satellite)
```

Comment:
Is this procedure only applicable to Satellite? The description and text below both use 'Katello'. But if this is something a user of Katello can/should also be having run on a Foreman then we need to update the confines.